### PR TITLE
Create logger middleware for dev env

### PR DESCRIPTION
**Pull request by dkryvi**
#4

### CODE REVIEW

File: app/server.js [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 const nconf = require('nconf');
 const globSync = require('glob').sync;
+const morgan = require('morgan');
 
 const { errorHandler, notFound } = require('./middleware/error_handler');
 
 const initApp = () => {
+  const env = process.env.NODE_ENV || 'dev';
+  if (env === 'dev') {
+    app.use(morgan('dev'));
```
Comment (akosenko) [inline app/server.js]: replace 'dev' with corresponding parameter

**Review by akosenko — CHANGES_REQUESTED** [review]

File: app/server.js (hunk 1 above)
Comment (dkryvi) [reply app/server.js]: `dev` it's one of middleware type. See [here](https://github.com/expressjs/morgan#dev)

Comment (akosenko) [reply app/server.js]: app.use(morgan(env)) - is not correct?

Comment (dkryvi) [reply app/server.js]: Morgan doesn't have `prod` or `test` types

Comment (akosenko) [reply app/server.js]: well, I think you see only this line... bad...
if (env === 'dev') {
   app.use(morgan(env));
}
answer the question: how prod or test type will pass in if block?

Comment (dkryvi) [reply app/server.js]: I think that we should use middleware log only in dev. Isn't it?

Comment (akosenko) [reply app/server.js]: yep

Comment (dkryvi) [reply app/server.js]: That's why I check environment only for dev

Comment (akosenko) [reply app/server.js]: please change code from `app.use(morgan('dev'));` to `app.use(morgan(env));`

Comment (dkryvi) [reply app/server.js]: [Done](https://github.com/inst-event-emitter/eventemitter-backend/pull/8/commits/b997d989f2f08098df989c881b1efaf7cf89dd23#diff-f53e59f241bb62f0261e245350683e07R11)

File: app/server.js [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
 const globSync = require('glob').sync;
+const morgan = require('morgan');
 
 const { errorHandler, notFound } = require('./middleware/error_handler');
 
 const initApp = () => {
+  const env = process.env.NODE_ENV || 'dev';
+  if (env === 'dev') {
+    app.use(morgan(env));
+  }
```
Comment (akosenko) [inline app/server.js:12]: it'll be great if we implement the following feature:
https://github.com/expressjs/morgan#log-file-rotation

**Review by akosenko — APPROVED** [review]

File: app/server.js (hunk 2 above)
Comment (dkryvi) [reply app/server.js:12]: I create a ticket for it

Comment (akosenko) [reply app/server.js:12]: I've created an issue

**Pull request merged**